changes/ocd-3517.sql

```diff
@@ -0,0 +1,42 @@
+DROP TABLE IF EXISTS openchpl.certification_criterion_attribute;
+
+CREATE TABLE openchpl.certification_criterion_attribute (
+  id                 bigserial NOT NULL,
+  criterion_id       bigint NOT NULL,
+  svap               bool NOT NULL DEFAULT false,
+  creation_date      timestamp NOT NULL DEFAULT NOW(),
+  last_modified_date timestamp NOT NULL DEFAULT NOW(),
+  last_modified_user bigint NOT NULL,
+  deleted            bool NOT NULL DEFAULT false,
+  CONSTRAINT certification_criterion_attribute_pk PRIMARY KEY (id),
+  CONSTRAINT certification_criterion_id_fk FOREIGN KEY (criterion_id)
+        REFERENCES openchpl.certification_criterion (certification_criterion_id)
+        MATCH SIMPLE ON UPDATE NO ACTION ON DELETE RESTRICT
+);
+
+CREATE TRIGGER certification_criterion_attribute_audit AFTER INSERT OR UPDATE OR DELETE on openchpl.certification_criterion_attribute FOR EACH ROW EXECUTE PROCEDURE audit.if_modified_func();
+CREATE TRIGGER certification_criterion_attribute_timestamp BEFORE UPDATE on openchpl.certification_criterion_attribute FOR EACH ROW EXECUTE PROCEDURE openchpl.update_last_modified_date_column();
+
+INSERT INTO openchpl.certification_criterion_attribute (criterion_id, svap, last_modified_user) VALUES (21, true, -1);
```

Comment:
If this runs twice, will it insert multiple rows allowing a given criteria to have SVAP? Isn't that the same issue we ran into with the dummy-data insert scripts? I know it isn't likely to run twice in PROD, but we have a policy of not doing this kind of insert in general, right?

Reply:
The first line in this script drops the table, so we won't have dups. 

We were getting dups with the svap stuff, because the creation of the table was in the ocd-????.sql file and the creation of the data was in the dummy-data.sql file.  So, when the ocd-???? file went away, we were only running the inserts with the drop table that was in the ocd-????.sql table.